python/ray/test/test_ray_init.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pytest
+import redis
+
+import ray
+
+
+class TestRedisPassword(object):
```

Comment:
I *think* the canonical way to write pytest tests is to not use classes and to instead use fixtures, e.g., instead of the `setup`/`teardown`, you can do something like the following.

https://github.com/ray-project/ray/blob/fcdca6de181a7272003a9ddfab27337f4f939373/test/failure_test.py#L29-L35